extensions/media-preview/media/imagePreview.css

```diff
@@ -31,20 +32,20 @@ body img {
 	box-sizing: border-box;
 }
 
-.container.image img {
+.container.image .transparency-grid {
 	padding: 0;
 	background-position: 0 0, 8px 8px;
 	background-size: 16px 16px;
 	border: 1px solid var(--vscode-imagePreview-border);
```

Comment:
`--vscode-imagePreview-border` isn't actually defined because I think it was inadvertently removed in https://github.com/microsoft/vscode/issues/122917 so this border wasn't even rendering before this PR.

Anyway, I've kept it on the grid because otherwise it'd get zoomed in as well.